style/style.css

```diff
@@ -0,0 +1,560 @@
+  background-repeat: no-repeat;
+  background-size: 85px;
+  padding: 15px;
+  object-fit: cover;
+}
+
+.presentation {
+  display: flex;
+  flex-direction: column;
+}
+
+.speakers hr {
+  width: 9%;
+  border: 1px solid  rgb(24, 135, 209);
+}
+
+.speaker-wrap p {
+  color: black;
+}
+
+.speaker-wrap hr {
+  border: 1px solid rgb(179, 175, 175);
+}
+
+/* .speakers .more {
+  text-align: center;
+  width: 80%;
+  padding: 20px;
+  margin: 10px;
+  border: 2px solid rgb(156, 151, 151);
+  color: rgb(68, 67, 67);
+}
+
+.more img {
+  width: 25px;
+  margin-left: 3px;
+} */
+
+.partner {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  border: 2px solid wheat;
+  background-color: #434344;
+  margin-top: 5%;
+  align-items: center;
+  justify-content: center;
+}
+
+.partner h2 {
+  text-align: center;
+  color: rgb(82, 155, 240);
+  margin-top: 3%;
+  font-weight: bolder;
+}
+
+.partner hr {
+  margin-bottom: 5%;
+  font-weight: bolder;
+}
+
+.partner-corporation {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+  margin: 2% 0;
+  padding: 3%;
+}
+
+.logo-rights {
+  display: flex;
+  flex-direction: row;
+  margin: 7%;
+}
+
+.logo-rights img {
+  width: 45%;
+}
+
+.logo-rights p {
+  color: #434344;
+  width: 60%;
+  height: 50px;
+  margin-top: 15%;
+  font-weight: bolder;
+}
+
+/* media query */
+
+@media all and (min-width: 768px) {
+  .top {
+    background: rgb(37, 36, 36);
+    display: flex;
+    align-items: center;
+  }
+
+  ul {
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+  }
+
+  .top ul {
+    list-style: none;
+    width: 10%;
+    margin-right: 10%;
+    color: rgb(255, 255, 255);
+  }
+
+  li a {
+    width: 20%;
+    font-weight: 600;
+    font-size: 1.2rem;
+    text-decoration: none;
+    color: #344563;
+    transition: 0.4s;
+  }
+
+  .top a {
+    color: rgb(255, 255, 255);
+    cursor: pointer;
+  }
+
+  .headline {
+    margin: 0;
+    padding: 0;
+  }
+
+  .navbar {
+    display: flex;
+  }
+
+  .navbar i {
+    display: none;
+  }
+
+  #logo {
+    width: 35%;
+  }
+
+  .desktop-menu {
+    display: flex;
+    width: 80vw;
+    justify-content: space-around;
+  }
+
+  .desktop-menu img {
+    width: 50%;
+  }
+
+  li {
+    list-style: none;
+    margin: 6%;
+  }
+
+  li a:hover {
+    color: rgb(69, 191, 221);
+  }
+
+  .compaign {
+    padding: 10px;
+    border: 2px solid rgb(35, 59, 122);
+  }
+
+  .compaign:hover {
+    border: 2px solid rgb(18, 208, 233);
+  }
+
+  .tittle {
+    margin: 7%;
+  }
+
+  .tittle h3 {
+    font-size: 2rem;
+  }
+
+  .tittle h1 {
+    font-size: 3.5rem;
+    width: 48%;
+  }
+
+  .headline-desc {
+    width: 35%;
+    margin: 5%;
+    padding: 0;
+  }
+
+  .date {
+    margin: 0 9%;
+  }
+
+  .date h3 {
+    font-size: 2.5rem;
+  }
+
+  .venue {
+    font-size: 1.2rem;
+  }
+
+  .container {
+    margin-top: 4%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    height: 60vh;
+  }
```

Comment:
- [ ] Your 'Main Program' section is great, but on screen sizes with larger heights this section creates a large whitespace at the bottom due to the `vh` value on the `height` attribute making a bad user experience. This attribute may not be necessary, so kindly remove it, the height of the section will be determined by the height of the items inside.
 - Your design:
 - 
<img width="965" alt="image" src="https://user-images.githubusercontent.com/61919617/203773164-e89c18b7-a823-4135-9275-6f48741d4e12.png">